frameworks/kafka/tests/test_sanity.py

```diff
@@ -47,20 +47,10 @@ def test_service_health():
 @pytest.mark.sanity
 def test_endpoints_address():
     foldered_name = sdk_utils.get_foldered_name(config.SERVICE_NAME)
+    endpoints = sdk_networks.wait_for_endpoint_info(
+        config.PACKAGE_NAME, foldered_name, config.DEFAULT_TASK_NAME
```

Comment:
I feel that using `"broker"` here instead of `config.DEFAULT_TASK_NAME` is clearer.

Reply:
Can Kafka have any other task names in general? Otherwise, I would replace all the occurrences and remove the variable in `config.py`.

Reply:
Kafka only has `broker` tasks: https://github.com/mesosphere/dcos-commons/blob/master/frameworks/kafka/src/main/dist/svc.yml#L31

Feel free to remove this variable.